### ISSUE TITLE
Support Spring Beans

### ISSUE THREAD

**Issue by varpa89** [mplushnikov/lombok-intellij-plugin#586]
When use spring with lombok there is no such features support like

- Spring Beans Dependencies Diagram
- Navigation to autowired dependencies

https://www.jetbrains.com/help/idea/spring-support.html

Is it possible to support them?

**Comment by noviodelangel** [comment]
Yes please! It would be so useful

**Comment by CC007** [comment]
Yes, for me especially when I create private final fields for the beans and use a `@RequiredArgsConstructor` it would be nice to have the bean link on the field.
